src/main/java/com/quiz/IQuestion.java

```diff
@@ -0,0 +1,9 @@
+package com.quiz;
+
+public interface IQuestion {
```

Comment:
If you want this IQuestion thing so desperately, you can name it IBaseQuestion for example and the child class to be something like PersonQuestion